### ISSUE TITLE
Puppet agent 7.29.0 break the module

### ISSUE THREAD

**Issue by elfranne** [voxpupuli/puppet-keepalived#314]
TLDR: upgrade Puppet agent to version >7.29.1

This is not so much of a bug report but more a note for search engines or other people stumbling upon the same error.

```log
Puppet Language validation logged 4 errors. Giving up (file: /etc/puppetlabs/code/environments/production/modules/keepalived/manifests/global_defs.pp) on node ha.example.net```
```

You can get more info:
```shell
puppet parser validate /etc/puppetlabs/code/environments/production/modules/keepalived/manifests/global_defs.pp
Error: The parameter '$bfd_priority' must be a literal type, not a Puppet::Pops::Model::AccessExpression (file: /etc/puppetlabs/code/environments/production/modules/keepalived/manifests/global_defs.pp, line: 118, column: 30)
Error: The parameter '$checker_priority' must be a literal type, not a Puppet::Pops::Model::AccessExpression (file: /etc/puppetlabs/code/environments/production/modules/keepalived/manifests/global_defs.pp, line: 119, column: 30)
Error: The parameter '$vrrp_priority' must be a literal type, not a Puppet::Pops::Model::AccessExpression (file: /etc/puppetlabs/code/environments/production/modules/keepalived/manifests/global_defs.pp, line: 120, column: 30)
Error: The parameter '$max_auto_priority' must be a literal type, not a Puppet::Pops::Model::AccessExpression (file: /etc/puppetlabs/code/environments/production/modules/keepalived/manifests/global_defs.pp, line: 129, column: 29)
```

More info about the issue here : https://github.com/puppetlabs/puppet/pull/9269

You will also be required to either restart the puppet server or:

```shell
curl -i --cert $(puppet config print hostcert) \
--key $(puppet config print hostprivkey) \
--cacert $(puppet config print cacert) \
-X DELETE \
https://$(puppet config print server):8140/puppet-admin-api/v1/environment-cache?environment=production
```

Thanks @bastelfreak for pointing into the right direction.
